### PR TITLE
Add pagination and cleanup code

### PR DESCRIPTION
**Pull request by Leblyad**

### CODE REVIEW

File: ServicesAPI/Infrastructure/Repository/UserClasses/ServiceRepository.cs [hunk 1]
```diff
@@ -24,7 +25,14 @@ public async Task DeleteServiceAsync(Service service)
...
         }
 
-        public async Task<IEnumerable<Service>> GetAllServicesAsync(bool trackChanges) =>
+        public async Task<IEnumerable<Service>> GetAllServicesAsync(ServiceParameters serviceParameters, bool trackChanges) =>
+            serviceParameters.PageNumber > 0
+            && serviceParameters.PageSize > 0 ?
+            await FindAll(trackChanges)
+            .Skip((serviceParameters.PageNumber - 1) * serviceParameters.PageSize)
+            .Take(serviceParameters.PageSize)
+            .Include(serviceCategory => serviceCategory.ServiceCategory)
```
[the remark this answers is not in the document]
Comment (kseniya-kalita) [reply ServicesAPI/Infrastructure/Repository/UserClasses/ServiceRepository.cs:34]: Please rename serviceCategory, it's another entity.

File: ServicesAPI/Infrastructure/Repository/UserClasses/ServiceCategoryRepository.cs [hunk 2]
```diff
@@ -24,11 +25,19 @@ public async Task DeleteServiceCategoryAsync(ServiceCategory serviceCategory)
             await RepositoryContext.SaveChangesAsync();
         }
 
-        public async Task<IEnumerable<ServiceCategory>> GetAllServiceCategoriesAsync(bool trackChanges) =>
+        public async Task<IEnumerable<ServiceCategory>> GetAllServiceCategoriesAsync(ServiceCategoryParameters serviceCategoryParameters, bool trackChanges) =>
```
Comment (kseniya-kalita) [inline ServicesAPI/Infrastructure/Repository/UserClasses/ServiceCategoryRepository.cs:28]: Please add default value to track changes (as in the interface).

**Review by kseniya-kalita — CHANGES_REQUESTED** [review]

File: ServicesAPI/Infrastructure/Repository/UserClasses/ServiceCategoryRepository.cs (hunk 2 above)
Comment (Leblyad) [reply ServicesAPI/Infrastructure/Repository/UserClasses/ServiceCategoryRepository.cs]: Done

**Review by kseniya-kalita — APPROVED** [review]
Looks good!

**Pull request merged**